### ISSUE TITLE
Auto refresh when updating doc doesn't work

### ISSUE THREAD

**Issue by nassimBouazabia** [docker/docker.github.io#8339]
### Problem description
May be I'm doing something wrong but I followed the Readme to commit a change locally but the pages are not updated.
I runed `docker-compose up` and the documentation is well available on localhost:4000, but when I can't see any change when updating documentation.
I also tried with test.md file and deleting cahe, the changes are not applied

### Problem location

localhost:4000

### Project version(s) affected

* Engine 
ENV ENGINE_SVN_BRANCH="branches/18.09.x"
ENV ENGINE_BRANCH="18.09.x"

* Distribution 
ENV DISTRIBUTION_SVN_BRANCH="branches/release/2.6"
ENV DISTRIBUTION_BRANCH="release/2.6"

**Comment by bermudezmt** [comment]
@nassimBouazabia I've added another way for you to test your changes. Please check out step 2 to build the site locally via Jekyll and step 3 of https://github.com/docker/docker.github.io#staging-the-docs to build and run a Docker container of your working copy of the site?

Please let me know if you still have questions.
